src/Cron_Event_Command.php

```diff
@@ -269,6 +269,47 @@ public function run( $args, $assoc_args ) {
 		WP_CLI::success( sprintf( $message, $executed ) );
 	}
 
+	/**
+	 * Unchedules cron event on specific hook.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <hook>
+	 * : The hook name.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Unschedule a cron event on given hook.
+	 *     $ wp cron event unschedule cron_test
+	 *     Success: Unscheduled 2 events with hook 'cron_test'.
+	 */
+	public function unschedule( $args, $assoc_args ) {
+
+		$hook = $args[0];
+
+		if ( Utils\wp_version_compare( '4.9.0', '<' ) ) {
+			WP_CLI::error( "The 'wp_unschedule_hook' function was only introduced in WordPress 4.9.0." );
+		}
+
+		$unscheduled = wp_unschedule_hook( $hook );
+
+		if ( empty( $unscheduled ) ) {
+			$message = 'Event not unscheduled.';
+
+			// If 0 event found on hook.
+			if ( 0 === $unscheduled ) {
+				$message = "No event found with hook '%1\$s'.";
+			}
+
+			WP_CLI::error( sprintf( $message, $hook ) );
+
+		} else {
+			$message = ( 1 === $unscheduled ) ? "Unscheduled event with hook '%2\$s'." : "Unscheduled %1\$d events with hook '%2\$s'.";
```

Comment:
Could we update this usage and use `_n` instead of conditional check here?
Count should be included for singular event as well, below snippet should work with that.
```suggestion
			WP_CLI::success(
				sprintf(
					'Unscheduled %1$d %2$s with hook %3$s.',
					$unscheduled,
					_n( 'event', 'events', $unscheduled )
				)
			);
```

Reply:
Done.